### ISSUE TITLE
question about log 

### ISSUE THREAD

**Issue by perfe1983** [mesosphere/marathon-lb#339]
hello

in our infrastracture we use marathon-lb on DC/OS cluster intalled on CoreOS server;
we configure docker with the option --log-driver="journald" and then we send journal log using rsyslog to an a remote log collector..

How is the best way to configure haproxy log?

tks

**Comment by brndnmtthws** [comment]
For the sake of others, there's documentation for this on the wiki: https://github.com/mesosphere/marathon-lb/wiki#enabling-haproxy-logging

**Comment by mhausenblas** [comment]
@brndnmtthws not so fast please. We were there (see also out discussion on #networking channel). The docs do not explain how to configure MLB in DC/OS. Can you clarify this please? I tried it myself and failed to configure it via `dcos package install --options=options.json marathon-lb`. To me it looks like this is not implemented in the Universe and there should be an issue for this.

**Comment by brndnmtthws** [comment]
That's correct, it's not an in the package. The reason for this is that the logs wouldn't go anywhere useful. It requires a few more steps to configure the host logging so that everything sent to `/dev/log` goes somewhere. I don't think that's within the scope of DC/OS, and it isn't for me to decide either. Furthermore, HAProxy is configured with the logging off by default, because HAProxy's logging isn't very useful most of the time (it's just a lot of noise).

Happy to leave this open though.

**Comment by mimmus** [comment]
Log is always useful: following calls from a client to container along the full path is an invaluable help

**Comment by brndnmtthws** [comment]
Certainly, but the haproxy logs don't really tell you much that tcpdump doesn't.

**Comment by perfe1983** [comment]
It is true with tcpdump you can trace all network traffic but the log is easier to create alarms and build kpi

**Comment by lvyc7chinaunicom** [comment]
@brndnmtthws hello, how do you open HAProxy log , and ,how to view the HAProxy log in docker?
can you tell me ?
thinks!

**Comment by brndnmtthws** [comment]
@lvyc7chinaunicom I don't work for Mesosphere anymore, but you can hire me as a consultant if you'd like.
